resources/zib/zib-LaboratoryTestResult.xml

```diff
@@ -0,0 +1,359 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">

```

Comment:
Missing explanation about why we don't map NL-CM:13.1.30 (InterpretationMethod) (see excel sheet)

Reply:
Explanation is added on the profile root.